ui/tests/acceptance/secrets/backend/aws/aws-configuration-test.js

```diff
@@ -231,6 +230,7 @@ module('Acceptance | aws | configuration', function (hooks) {
       createConfig(this.store, path, type); // create the aws root config in the store
       await click(SES.configTab);
       for (const key of expectedConfigKeys(type)) {
+        if (key === 'Secret key') return; // secret-key is not returned by the API
```

Comment:
I think `return` will stop overall execution of the function. 

Maybe `continue`? 
```suggestion
        if (key === 'Secret key') continue; // secret-key is not returned by the API
```